SConstruct

```diff
@@ -436,6 +439,12 @@ else:
     # Disable assert() for production targets (only used in thirdparty code).
     env_base.Append(CPPDEFINES=["NDEBUG"])
 
+    # SCU builds currently use too much compiler memory
+    # in release builds, so disallow except in DEV builds.
+    if env_base["use_scu"]:
+        print("WARNING: SCU build flag ignored in non-DEV builds.")
+        env_base["use_scu"] = False
```

Comment:
For benefit of @Calinou , if you want to try release build locally, just comment out these lines, and let us know if it works at all / timings. For me, even when I got it to work, it was slower because RAM / swapping became a problem.

If it is viable we could easily add another command line variation to allow it in release builds as well as DEV (and maybe use this to increase the chunking too).